### ISSUE TITLE
Allow option to provide own clients

### ISSUE THREAD

**Issue by Salakar** [deepstreamIO/deepstream.io-msg-redis#6]
Would be good to have the option to provide our own redis clients for pubsub for this connection, i.e

``` javascript
{
    subscriber: MySubClient,
    publisher: MyPubClient
}
```

**Comment by WolframHempel** [comment]
That shouldn't be a problem, just to make sure though: `MySubClient` would be an instance of the client returned by `redis.createClient( options.port, options.host );`, right?

**Comment by Salakar** [comment]
Yes, both would be, exactly like https://www.npmjs.com/package/socket.io-redis#adapteropts allows you to do. (in their instance, pubClient and subClient options)

Just allows people using clients other than `node-redis` to use it. I for example use `ioredis` as it supports clustering.
